packages/web/package-lock.json

```diff
@@ -1,11396 +1,8 @@
 {
   "name": "gcds-components",
   "version": "0.1.15",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
How come all packages are being removed from here? Was that by mistake?

Reply:
Probably, what does that do? I installed the font awesome package. That's the only thing I can see.